libflux/src/flux/semantic/builtins.rs

```diff
@@ -117,7 +117,12 @@ pub fn builtins() -> Builtins<'static> {
                 // https://github.com/influxdata/flux/issues/2250
                 "json" => "forall [t0] where t0: Row (?json: string, ?file: string) -> [t0]",
                 "databases" => r#"
-                    forall [] () -> [{
+                    forall [] (
```

Comment:
Huh. How did we miss this?

Reply:
They're new parameters that were added in this PR.